src/core/providers/LocalSamAudioWorkerProvider.js

```
@@ -0,0 +1,229 @@
+      for (let c = 0; c < audio.length; c++) s += audio[c][i] || 0;
+      out[i] = s / audio.length;
+    }
+    return out;
+  }
+  throw new TypeError('[VIP][sam-worker] audio must be Float32Array or Float32Array[]');
+}
+
+export function float32ToBase64(f32) {
+  const bytes = new Uint8Array(f32.buffer, f32.byteOffset, f32.byteLength);
+  if (typeof Buffer !== 'undefined') {
+    return Buffer.from(bytes).toString('base64');
```

ESLint

[error] 'Buffer' is not defined.

(no-undef)